activate.sh

```diff
@@ -7,27 +7,27 @@ autoenv_init() {
 
 	_mountpoint="$(df -P "${PWD}" | tail -n 1 | awk '{ for(i=6; i<NF; i++) printf "%s",$i OFS; if(NF) printf "%s",$NF; printf ORS}')"
```

Comment:
@morbeo Yep, I only change old command substitution to new form, don't check the logic in command